style/ComponentNative/ToggleControl.mcss

```diff
@@ -46,4 +46,5 @@
   display: none;
   position: relative;
   border-top: 1px solid black;
+  min-height: 100px;
```

Comment:
Do you get the panel to grow based on its content?
I would even go lower like 1rem